Cargo.toml

```diff
@@ -2,8 +2,10 @@
 name = "send-with-sendgrid-and-rust"
 version = "0.1.0"
 authors = ["Matthew Setter"]
+edition = "2018"
```

Comment:
this was previously missing, which resulted in the "missing dependency" errors

Reply:
I totally overlooked that in the documentation. Thank you.